Added readme file

neliudochka opened the pull request:
Added readme file. I will update it as the project grows.

AKushch1337 commented on README.md:
```diff
@@ -0,0 +1,20 @@
+# Steganography
+
+
+Steganography is a a mobile application, with which you can encode messages in the photos you upload, and make reverse action: decode messages from the uploaded photos.  
+
+You can select images from your gallary and encdode in them your massages. Also, you can save them to you device.  
+
```
The second paragraph repeats the info from the first one, rewrite it please.

AKushch1337 commented on README.md:
````diff
@@ -0,0 +1,20 @@
+  
+Please, comment it, if you have questions or propositions on how to make this project better.  
+
+
+## Installation
+Clone this repository and import into **Android Studio**
+```bash
+git clone git@github.com:neliudochka/steganography.git
+```
+Build it there
````
Expand on the topic on how to build or run your app. Read how to build your project with gradle: [link](https://developer.android.com/build/building-cmdline). In short, you can run 
```gradle
./gradlew build
```
and it will fully build the project and run unittests.

AKushch1337 reviewed (changes requested):
Here are some ideas for your README:
- Write about the tools you used for creating this project(IDE, Language, etc.)
- Add some screenshots of your app
- Just as a reminder(as your project is in progress): when you finish creating your project build an APK and add it to the release page in your repo, so that the user can test the app without actually building it himself

neliudochka commented:
Updated readme file.

Expanded installation section. Added Build, Run, Project status, Contact, Technologies & Tools Used sections. Fixed project description.

neliudochka commented:
Force push because I am stupid and did not check commits messages
